spec-main/api-browser-window-spec.ts

```diff
@@ -1695,12 +1695,30 @@ describe('BrowserWindow module', () => {
     });
 
     it('Allows setting a transparent window via CSS', async () => {
-      const appPath = path.join(__dirname, 'fixtures', 'apps', 'background-color-transparent');
+      const display = screen.getPrimaryDisplay();
+      const TRANSPARENT_BG_COLOR = '#515152';
```

Comment:
My understanding is that this is the resulting color of the vibrancy window backdrop composited with the desktop background. We may need to place a window behind this one with a solid color to have consistent results in the test.

I used the chroma key color for this purpose in other tests:
https://github.com/electron/electron/blob/b888d9cd17d41a7b7a2e2bf61429fc1f5d3c94e4/spec-main/api-browser-window-spec.ts#L4904-L4909

Reply:
@samuelmaddock either way here it's going to be a bit of an arbitrary color since vibrancy darkens the window - is that what you would prefer? I'd then basically add a bgwindow and check for the fg window being `#244a24` since that's what itd come out to with vibrancy